src/main.rs

```diff
@@ -34,15 +35,26 @@ mod operations;
 mod s3_client;
 mod validated_json;
 
+/// S3 Active Storage Proxy command line interface
+#[derive(Debug, Parser)]
+struct CommandLineArgs {
+    #[arg(long, default_value = "0.0.0.0", env = "S3_ACTIVE_STORAGE_HOST")]
+    host: String,
+    #[arg(long, default_value = "8080", env = "S3_ACTIVE_STORAGE_PORT")]
+    port: String,
```

Comment:
Let's make it a u16 to get some better input validation.

```suggestion
    #[arg(long, default_value = 8080, env = "S3_ACTIVE_STORAGE_PORT")]
    port: u16,
```

Reply:
Or there's this: https://github.com/clap-rs/clap-port-flag